rrule/tests/properties.rs

```diff
@@ -0,0 +1,80 @@
+mod common;
+
+use chrono::TimeZone;
+use chrono_tz::UTC;
+use rrule::{Frequency, NWeekday, RRuleProperties, Weekday};
+use std::str::FromStr;
+
+fn get_obj_cases() -> Vec<RRuleProperties> {
+    vec![
+        RRuleProperties {
+            freq: Frequency::Yearly,
+            count: Some(3),
+            ..Default::default()
+        },
+        RRuleProperties {
+            freq: Frequency::Weekly,
+            interval: 5,
+            by_weekday: vec![
+                NWeekday::Nth(-2, Weekday::Mon),
+                NWeekday::Every(Weekday::Fri),
+            ],
+            ..Default::default()
+        },
+        RRuleProperties {
+            freq: Frequency::Weekly,
+            until: Some(UTC.ymd(1999, 4, 4).and_hms(11, 0, 0)),
+            by_weekday: vec![NWeekday::Every(Weekday::Tue), NWeekday::Every(Weekday::Wed)],
+            ..Default::default()
+        },
+        RRuleProperties {
+            freq: Frequency::Weekly,
+            until: Some(UTC.ymd(2019, 1, 1).and_hms(23, 0, 0)),
+            by_weekday: vec![NWeekday::Every(Weekday::Tue), NWeekday::Every(Weekday::Wed)],
+            ..Default::default()
+        },
+    ]
+}
+
```

Comment:
I thought all filenames are the same as the file that we actually test.
So... because of that I put `rrule` serde parts in the `rrule` test file. and so on.